src/DynamoCoreWpf/ViewModels/PackageManager/PackageManagerSearchViewModel.cs

```diff
@@ -263,8 +263,9 @@ public PackageManagerSearchViewModel(PackageManagerClientViewModel client) : thi
         {
             this.PackageManagerClientViewModel = client;
             PackageManagerClientViewModel.Downloads.CollectionChanged += DownloadsOnCollectionChanged;
```

Comment:
I don't think we clean these up from anywhere - probably we should do it in the PackageManagerSerachView.Unloaded framework event.

Reply:
Could this be added here in the `OnClosing` handler for the `PackageManagerSearchView`: https://github.com/DynamoDS/Dynamo/blob/405cac2930f0f72e5569deac241b6e4e5b3442d6/src/DynamoCoreWpf/Views/PackageManager/PackageManagerSearchView.xaml.cs#L26

Reply:
`OnClosing` seems a good spot